README.md

```diff
@@ -112,6 +112,32 @@ Now you can compile your SASS files running:
 ```
 Which will take all the `scss` files inside the `styles/` folder and create the `style.css` file that is consumable by Zendesk Guide.
 
+
+# Quick Setup Recall (Running and testing theme locally)
+
+NOTE: This guide is to quickly setup and run the theme locally. To know more about how to customize the theme, See all the details above.
+
+1. Create a trial account on https://www.zendesk.com/. This will let you create a subdomin Zendesk account that you can manage.
+2. Make sure to opt-in for Guide integration once your trial account setup is done.
+3. Create API keys for running the server locally and previewing them on your subdomain. `<YOUR_ZENDESK_DOMAIN>/admin/apps-integrations/apis/zendesk-api/settings`.
+4. Enable Help Center for your account through `<YOUR_ZENDESK_DOMAIN>/hc/admin/general_settings`
+5. Install [ZAT](https://developer.zendesk.com/documentation/apps/zendesk-app-tools-zat/installing-and-using-zat/) Or [ZAT CLI](https://support.zendesk.com/hc/en-us/articles/4408822095642-Previewing-theme-changes-locally)
```

Comment:
If we only install the ZAT CLI, we cannot build assets as it throws errors when we run `./bin/compile.rb`
```suggestion
5. Install [ZAT](https://developer.zendesk.com/documentation/apps/zendesk-app-tools-zat/installing-and-using-zat/) and [ZAT CLI](https://support.zendesk.com/hc/en-us/articles/4408822095642-Previewing-theme-changes-locally)
```

Reply:
Or maybe we should just exclude the ZAT CLI part as it can get a bit confusing. It has a different command for previewing the theme `zcli themes:preview`

Reply:
ZAT CLI is the recommended option for the future development of Zendesk themes. I think we should keep it. I've updated with the suggestion.